app.js

```diff
@@ -6,20 +6,29 @@
 // a number of minutes as an argument and returns
 // the number of seconds
 
-function convertToSeconds() {}
-
+function convertToSeconds(minutes) {
+    return minutes * 60;
+}
 console.log(`convertToSeconds(10) === 600 : ${convertToSeconds(10) === 600}`);
 
 // 2. areaOfTriangle
 // Write a function called areaOfTriangle that takes in the base and height
 // of a triangle and returns its area
-function areaOfTriangle() {}
+function areaOfTriangle(base, height) {
+    return base * height / 2;
+}
 console.log(`areaOfTriangle(5, 10) === 25 : ${areaOfTriangle(5, 10) === 25}`);
 
 // 3. lessThan100
 // Given 2 numbers, return true if their sum is less than 100
 // return false if it is greater than 100
-function lessThan100() {}
+function lessThan100(A, B) { 
+    if (A + B < 100) {
+        return true;
+    } else (A + B >= 100) 
```

Comment:
for this if-else statement, you actually don't need anything on line 28 except  
' } else { '